modules/nextflow/src/main/groovy/nextflow/fusion/FusionEnvProvider.groovy

```diff
@@ -39,9 +39,10 @@ class FusionEnvProvider {
         }
         // tags setting
         if( config.tagsEnabled() ) {
-            result.FUSION_TAGS = config
-                    .tagsPattern()
-                    .replace('[*]', "[${bean.outputFiles.join('|')}](nextflow.io/output=true),[*]")
+            def tagsPattern = config.tagsPattern()
+            if( bean.outputFiles )
+                tagsPattern = tagsPattern.replace('[*]', "[${bean.outputFiles.join('|')}](nextflow.io/output=true),[*]")
```

Comment:
I think that it may be more safe to use a place holder instead of `[*]` to decide where to add these dynamic tags. We control the default value but is also possible that the user defines his own tag patterns. I will do something like `[.command.*|.exitcode|.fusion.*](nextflow.io/metadata=true),[##PROCESS_OUTPUT##](nextflow.io/output=true),[*](nextflow.io/temporary=true)` and then you replace only this `##PROCESS_OUTPUT##` if exists.

Reply:
Yes, no file is going to match `[](nextflow.io/output=true)` tag.